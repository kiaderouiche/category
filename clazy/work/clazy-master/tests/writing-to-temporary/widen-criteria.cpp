struct Foo
{
    void move(int v) { }
};

Foo getFoo() { return Foo(); }

void test()
{
    getFoo().move(1);
}
