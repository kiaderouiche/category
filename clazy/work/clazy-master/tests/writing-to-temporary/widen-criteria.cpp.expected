writing-to-temporary/widen-criteria.cpp:10:5: warning: Call to temporary is a no-op: Foo::move [-Wclazy-writing-to-temporary]
